### ISSUE TITLE
Request is not authorized to perform this operation

### ISSUE THREAD

**Issue by makashu** [microsoft/AzureStorageExplorer#4559]
**Storage Explorer Version**: 1.14.2

Bug Description:
------------------- 
I am facing "Not Authorized to perform this operation" issue while trying to access artifacts stored in blob container via the Microsoft Storage Explorer(V1.14.2) installed on my Windows 10 laptop. I am the Owner of the Azure subscription and still getting that issue while trying to browse to the artifacts in the Blob container via the Azure Storage Explorer. I do not get that issue while browsing the Blob container artifacts via the Azure browser Explorer(https://portal.azure.com/) using the same Azure subscription. 
The issue I am seeing is from today(June 15, 2021) only as I was able to access the artifacts via Microsoft Storage Explorer till yesterday(June 14, 2021). 
Can anyone please assist or help me on this?
![image](https://user-images.githubusercontent.com/17219766/122076709-01731b80-cdf3-11eb-9ef5-8c634361fa49.png)

Steps to Reproduce:
-----------------------
1. Launch Azure Storage explorer(v1.14.2) using subscription owner account
2. Browser to the Blob Container and try to expand it would report error message "This request is not authorized to perform this operation.\nRequestId:2315a576-601e-0091-6af6-6167e5000000\n"

Please note that access to Blob-Container via the Azure portal works absolutely fine https://portal.azure.com/

Actual Experience:
--------------------
![image](https://user-images.githubusercontent.com/17219766/122078602-9e828400-cdf4-11eb-9d5b-6f6efefc133d.png)

Expected Experience:
-----------------------
Should be able to read the artifacts uploaded in the Azure Storage blob container while browsing using the Azure Storage Explorer

Additional Context:
----------------------
Accessing it under the Company Owned Azure Subscription account.
![image](https://user-images.githubusercontent.com/17219766/122079308-31bbb980-cdf5-11eb-9ca8-b0957f80d40f.png)

**Comment by MRayermannMSFT** [comment]
@makashu likely something has changed with the storage account you are attempting to access.

Have you/anyone enabled firewall or restricted IPs for your storage account? Also, maybe try upgrading to the latest version of Storage Explorer?

**Comment by makashu** [comment]
Hi @MRayermannMSFT, thanks for your response.
I have checked with my cloud admin team and there seems to be no change on the firewall or the Storage Account Networking as I can see my IP address listed there.
One interesting thing that I realized that if I use the "Storage Explorer (preview)" using my account in https://portal.azure.com/ then I can see the container artifacts and no error message is seen, so it seems that portal explorer works and seems an issue while connecting from the Azure Storage Explorer(the https://management.azure.com/)
![image](https://user-images.githubusercontent.com/17219766/123109386-1b37e280-d433-11eb-8ed4-6a64fc755407.png)


Are there different API calls been made while browsing via the Azure Storage Explorer App versus the Azure Portal?

I am also going to check with latest Azure Storage Explorer but will need to get some approval from my company before we use the latest version.
Thanks!
